cmd/promtool/unittest.go

```diff
@@ -79,7 +79,7 @@ func ruleUnitTest(filename string) []error {
 	}
 
 	// Bounds for evaluating the rules.
-	mint := time.Unix(0, 0)
+	mint := time.Unix(0, 0).Add(unitTestInp.StartTimeOffset)
```

Comment:
This is only catching the evals, this should affect the series too.

Reply:
PS: While taking care for series, should make sure that the samples should be loaded till mint minus maximum lookback in any rule queries.